Cubical/Algebra/CommAlgebra/Instances/Initial.agda

```diff
@@ -0,0 +1,76 @@
+{-# OPTIONS --cubical --safe --no-import-sorts #-}
+module Cubical.Algebra.CommAlgebra.Instances.Initial where
+
+open import Cubical.Foundations.Everything
+open import Cubical.Foundations.HLevels
+
+open import Cubical.Data.Unit
+open import Cubical.Data.Sigma.Properties using (Σ≡Prop)
+
+open import Cubical.Algebra.CommRing
+open import Cubical.Algebra.Ring
+open import Cubical.Algebra.Algebra.Base using (IsAlgebraHom)
+open import Cubical.Algebra.CommAlgebra.Base
+
+private
+  variable
+    ℓ : Level
+
+module CommAlgebraExamples ((R , str) : CommRing ℓ) where
+
+  initial : CommAlgebra (R , str) ℓ
```

Comment:
Call this `initialCommAlgebra`? Same changes apply to other names in this module

Reply:
I made this module anonymous.